### ISSUE TITLE
[BUG/ISSUE] Wrong species mentioned in error message in state_chm_mod

### ISSUE THREAD

**Issue by chfite** [geoschem/geos-chem#416]
I noticed a small typo in state_chm_mod in the Init_HetInfo subroutine, where the error message for species HMML actually mentions the species HONIT. HONIT is the species before HMML in the subroutine which is probably why that typo was made, I assume. This was found in GC 12.9.2. 

### Issue

```
    N = Ind_('HMML')
    IF ( N > 0 ) THEN
       State_Chm%HetInfo%HMML%mId  = N
       State_Chm%HetInfo%HMML%MW_g = State_Chm%SpcData(N)%Info%MW_g
       State_Chm%HetInfo%HMML%K0   = State_Chm%SpcData(N)%Info%Henry_K0
       State_Chm%HetInfo%HMML%CR   = State_Chm%SpcData(N)%Info%Henry_CR
       State_Chm%HetInfo%HMML%TK   = 298.15_fp
    ELSE
       errMsg = &
        'HONIT is undefined, but is needed for heterogeneous chemistry!'
       CALL GC_Error( errmsg, RC, thisLoc )
       RETURN
    ENDIF
```

**Comment by lizziel** [comment]
Thanks @chfite, that is indeed an error. It will be fixed in the next version release. Thanks for reporting this!

**Comment by msulprizio** [comment]
This is now fixed in https://github.com/geoschem/geos-chem/commit/3b5192763ba36df20984fd0794afa5cde5ee67f3 and will be included in 13.0.0.
